ai_summarizer/admin.py

```
@@ -1,9 +1,15 @@
 from django.contrib import admin
 from .models import AISummary
 
+
+@admin.register(AISummary)
+class AISummaryAdmin(admin.ModelAdmin):
 @admin.register(AISummary)
 class AISummaryAdmin(admin.ModelAdmin):
+    # resource_type یک property نمایشی‌ه و برای list_display اوکیه
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)